packages/react-ui-core/src/Collapsible/Collapsible.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component, PropTypes } from 'react'
+
+export default class Collapsible extends Component {
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+    this.state = {
+      display: false,
+      toggleItems: this.props.toggleItems,
+    }
+  }
+  handleClick() {
+    this.props.handleClick(this)
+  }
+  render() {
+    const {
+      theme,
+      showableItems,
+      nonshowableItems,
+      id,
+      title,
+      checkboxClass,
+    } = this.props
+    const { toggleItems } = this.state
+    return (
+    <div id={id} className={this.props.collapsible}>
+        <p><strong>{title}</strong></p>
+        {this.props.showableItems}
+        <div className={this.state.display ? theme.show : theme.hide}>
+          {this.props.nonshowableItems}
+        </div>
+        <div className={theme.alignBottom}>
+          <a onClick={this.handleClick}>
+            <strong>{toggleItems}</strong>
+          </a>
+        </div>
+    </div>
+    )
+  }
+}
+Collapsible.defaultProps = {
```

Comment:
This is an acceptable style when doing functional components, but when creating an object (extending `Component` or `PureComponent`) we like to keep the `propTypes` and `defaultProps` in the object definition, near the top. [A propTypes example](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Form/Field.js#L15). [A defaultProps example](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Form/Field.js#L109).